src/identity/identity.ts

```diff
@@ -20,4 +20,8 @@ export class Identity {
 	isOrganisationManager() {
 		return this.hasRole('ORGANISATION_MANAGER')
 	}
+
+	isCshrReporter() {
+		return this.hasRole('CSHR_REPORTER')
+	}
```

Comment:
There are also ORGANISATION_REPORTER and PROFESSION_REPORTER roles that will be able to build reports. I think the ORGANISATION_MANAGER role allows someone to create and edit organisations rather than generate reports.

Reply:
Rob is correct @mick-jones

Reply:
2 new role checks have been added

isProfessionReporter() & isOrganisationReporter()